server/request_object.cc

```diff
@@ -729,10 +729,7 @@ void RequestObject::send_response(int code, std::string body) {
   if (!is_service_req_head) {
     s3_log(S3_LOG_INFO, request_id, "Sending response as: [code:%d] [%s]\n",
            code, body.c_str());
-  } else {
```

Comment:
tagging @knrajnambiar76 
Rajesh added this else part few days back.

Reply:
@amitwac2608 I asked @sachitanands  to remove it, some more optimization for health check requests